monica:latest not rebuilt?

It appears that for some reason the monica:latest tag was not rebuilt, despite the base image changing:

https://hub.docker.com/_/monica/tags / https://hub.docker.com/layers/library/monica/latest/images/sha256-9e4270cde2bbb51b07dafa3f8df6ddc9f7fdb2507fc03a993deddd12f5b20737?context=explore

latest was last updated 3 months ago, however the `php:8.1-apache` base image was last rebuilt a few hours ago:

https://hub.docker.com/layers/library/php/8.1-apache/images/sha256-e4912ed6c18ad4f7fbcd39d7c20b73f7e6e3215176882bde6ae492278f8ef3b9?context=explore

The build queue is still very full (https://doi-janky.infosiftr.net/) since all Debian images have been working though rebuilds since https://github.com/docker-library/official-images/pull/15395 was merged.

But it does look like the Debian-based versions of `monica` have been failing to build for a while (https://doi-janky.infosiftr.net/job/multiarch/view/images/view/monica/) and the error seems very familiar:

```console
checking for libmemcached location... configure: error: memcached support requires libmemcached. Use --with-libmemcached-dir=<DIR> to specify the prefix where libmemcached headers and library are located
ERROR: `/tmp/pear/temp/memcached/configure --with-php-config=/usr/local/bin/php-config --with-libmemcached-dir=no --with-zlib-dir=no --with-system-fastlz=no --enable-memcached-igbinary=no --enable-memcached-msgpack=no --enable-memcached-json=no --enable-memcached-protocol=no --enable-memcached-sasl=yes --enable-memcached-session=yes' failed
```

Aha! found it: https://github.com/docker-library/php/issues/1418; So, the `monica` Dockerfiles need to add `libssl-dev` to the `apt-get install` of dev dependencies (or switch to the bullseye-based PHP images).

--- a/library/monica
+++ b/library/monica
@@ -1,19 +1,19 @@
-# This file is generated via https://github.com/monicahq/docker/blob/1969cbd6d7dd410df84a18f539836ed4e0788324/generate-stackbrew-library.sh
+# This file is generated via https://github.com/monicahq/docker/blob/7337376d98838bc3ab2b7517da05a89e830d68b1/generate-stackbrew-library.sh
 Maintainers: Alexis Saettler <alexis@saettler.org> (@asbiin)
 GitRepo: https://github.com/monicahq/docker.git
 GitFetch: refs/heads/main
 
 Tags: 4.0.0-apache, 4.0-apache, 4-apache, apache, 4.0.0, 4.0, 4, latest
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
-Directory: apache
-GitCommit: 1969cbd6d7dd410df84a18f539836ed4e0788324
-
-Tags: 4.0.0-fpm-alpine, 4.0-fpm-alpine, 4-fpm-alpine, fpm-alpine
-Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, s390x
-Directory: fpm-alpine
-GitCommit: 1969cbd6d7dd410df84a18f539836ed4e0788324
+Directory: 4/apache
+GitCommit: 96189780c639d1511691a3151dd4d90d2f757ee3
 
 Tags: 4.0.0-fpm, 4.0-fpm, 4-fpm, fpm
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, mips64le, ppc64le, s390x
-Directory: fpm
-GitCommit: 1969cbd6d7dd410df84a18f539836ed4e0788324
+Directory: 4/fpm
+GitCommit: 96189780c639d1511691a3151dd4d90d2f757ee3
+
+Tags: 4.0.0-fpm-alpine, 4.0-fpm-alpine, 4-fpm-alpine, fpm-alpine
+Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, s390x
+Directory: 4/fpm-alpine
+GitCommit: 96189780c639d1511691a3151dd4d90d2f757ee3

--- a/test/tests/monica-apache-run/run.sh
+++ b/test/tests/monica-apache-run/run.sh
@@ -15,7 +15,7 @@ fi
 mysqlCid="$(docker run -d \
 	-e MYSQL_RANDOM_ROOT_PASSWORD=true \
 	-e MYSQL_DATABASE=monica \
-	-e MYSQL_USER=homestead \
+	-e MYSQL_USER=monica \
 	-e MYSQL_PASSWORD=secret \
 	"$dbImage")"
 trap "docker rm -vf $mysqlCid > /dev/null" EXIT
@@ -23,6 +23,10 @@ trap "docker rm -vf $mysqlCid > /dev/null" EXIT
 cid="$(docker run -d \
 	--link "$mysqlCid":mysql \
 	-e DB_HOST=mysql \
+	-e DB_CONNECTION=mysql \
+	-e DB_DATABASE=monica \
+	-e DB_USERNAME=monica \
+	-e DB_PASSWORD=secret \
 	"$image")"
 trap "docker rm -vf $cid $mysqlCid > /dev/null" EXIT
 
@@ -40,5 +44,5 @@ _request() {
 # Make sure that Apache is listening and ready
 . "$dir/../../retry.sh" --tries 30 '_request GET / --output /dev/null'
 
-# Check that we can request / and that it contains the pattern "Welcome" somewhere
-_request GET '/' | grep -i "Welcome" > /dev/null
+# Check that we can request / and that it contains the pattern "Monica" somewhere
+_request GET '/' | grep -i "Monica" > /dev/null

--- a/test/tests/monica-cli-mariadb10/run.sh
+++ b/test/tests/monica-cli-mariadb10/run.sh
@@ -15,7 +15,7 @@ fi
 mysqlCid="$(docker run -d \
 	-e MYSQL_RANDOM_ROOT_PASSWORD=true \
 	-e MYSQL_DATABASE=monica \
-	-e MYSQL_USER=homestead \
+	-e MYSQL_USER=monica \
 	-e MYSQL_PASSWORD=secret \
 	"$dbImage")"
 trap "docker rm -vf $mysqlCid > /dev/null" EXIT
@@ -23,6 +23,10 @@ trap "docker rm -vf $mysqlCid > /dev/null" EXIT
 cid="$(docker run -d \
 	--link "$mysqlCid":mysql \
 	-e DB_HOST=mysql \
+	-e DB_CONNECTION=mysql \
+	-e DB_DATABASE=monica \
+	-e DB_USERNAME=monica \
+	-e DB_PASSWORD=secret \
 	"$image")"
 trap "docker rm -vf $cid $mysqlCid > /dev/null" EXIT
 

--- a/test/tests/monica-cli-mysql8/run.sh
+++ b/test/tests/monica-cli-mysql8/run.sh
@@ -15,7 +15,7 @@ fi
 mysqlCid="$(docker run -d \
 	-e MYSQL_RANDOM_ROOT_PASSWORD=true \
 	-e MYSQL_DATABASE=monica \
-	-e MYSQL_USER=homestead \
+	-e MYSQL_USER=monica \
 	-e MYSQL_PASSWORD=secret \
 	"$dbImage")"
 trap "docker rm -vf $mysqlCid > /dev/null" EXIT
@@ -23,6 +23,10 @@ trap "docker rm -vf $mysqlCid > /dev/null" EXIT
 cid="$(docker run -d \
 	--link "$mysqlCid":mysql \
 	-e DB_HOST=mysql \
+	-e DB_CONNECTION=mysql \
+	-e DB_DATABASE=monica \
+	-e DB_USERNAME=monica \
+	-e DB_PASSWORD=secret \
 	"$image")"
 trap "docker rm -vf $cid $mysqlCid > /dev/null" EXIT
 

--- a/test/tests/monica-cli/run.sh
+++ b/test/tests/monica-cli/run.sh
@@ -15,7 +15,7 @@ fi
 mysqlCid="$(docker run -d \
 	-e MYSQL_RANDOM_ROOT_PASSWORD=true \
 	-e MYSQL_DATABASE=monica \
-	-e MYSQL_USER=homestead \
+	-e MYSQL_USER=monica \
 	-e MYSQL_PASSWORD=secret \
 	"$dbImage")"
 trap "docker rm -vf $mysqlCid > /dev/null" EXIT
@@ -23,6 +23,10 @@ trap "docker rm -vf $mysqlCid > /dev/null" EXIT
 cid="$(docker run -d \
 	--link "$mysqlCid":mysql \
 	-e DB_HOST=mysql \
+	-e DB_CONNECTION=mysql \
+	-e DB_DATABASE=monica \
+	-e DB_USERNAME=monica \
+	-e DB_PASSWORD=secret \
 	"$image")"
 trap "docker rm -vf $cid $mysqlCid > /dev/null" EXIT
 

--- a/test/tests/monica-fpm-run/run.sh
+++ b/test/tests/monica-fpm-run/run.sh
@@ -8,9 +8,9 @@ image="$1"
 # Build a client image with cgi-fcgi for testing
 clientImage="librarytest/monica-fpm-run:fcgi-client"
 docker build -t "$clientImage" - > /dev/null <<'EOF'
-FROM debian:stretch-slim
+FROM debian:buster-slim
 
-RUN set -x && apt-get update && apt-get install -y libfcgi0ldbl && rm -rf /var/lib/apt/lists/*
+RUN set -x && apt-get update && apt-get install -y --no-install-recommends libfcgi-bin && rm -rf /var/lib/apt/lists/*
 
 ENTRYPOINT ["cgi-fcgi"]
 EOF
@@ -25,7 +25,7 @@ fi
 mysqlCid="$(docker run -d \
 	-e MYSQL_RANDOM_ROOT_PASSWORD=true \
 	-e MYSQL_DATABASE=monica \
-	-e MYSQL_USER=homestead \
+	-e MYSQL_USER=monica \
 	-e MYSQL_PASSWORD=secret \
 	"$dbImage")"
 trap "docker rm -vf $mysqlCid > /dev/null" EXIT
@@ -33,6 +33,10 @@ trap "docker rm -vf $mysqlCid > /dev/null" EXIT
 cid="$(docker run -d \
 	--link "$mysqlCid":mysql \
 	-e DB_HOST=mysql \
+	-e DB_CONNECTION=mysql \
+	-e DB_DATABASE=monica \
+	-e DB_USERNAME=monica \
+	-e DB_PASSWORD=secret \
 	"$image")"
 trap "docker rm -vf $cid $mysqlCid > /dev/null" EXIT
 
@@ -60,5 +64,5 @@ fcgi-request() {
 # Make sure that PHP-FPM is listening and ready
 . "$dir/../../retry.sh" --tries 30 'fcgi-request GET /index.php' > /dev/null 2>&1
 
-# Check that we can request /register and that it contains the pattern "Welcome" somewhere
-fcgi-request GET '/index.php' register | grep -i "Welcome" > /dev/null
+# Check that we can request /register and that it contains the pattern "Monica" somewhere
+fcgi-request GET '/index.php' register | grep -i "Monica" > /dev/null

